lib/containers/urls.pm

```diff
@@ -57,52 +57,46 @@ sub get_suse_container_urls {
     my $dotversion = $version =~ s/-SP/./r;                    # 15 -> 15, 15-SP1 -> 15.1
     $dotversion = "${dotversion}.0" if $dotversion !~ /\./;    # 15 -> 15.0
 
-    my @image_names  = ();
-    my @stable_names = ();
+    my @untested_images = ();
+    my @official_images = ();
     if (is_sle(">=12-sp3", $version) && is_sle('<15', $version)) {
         my $lowerversion  = lc $version;
         my $nodashversion = $version =~ s/-sp/sp/ir;
         # No aarch64 image
         if (!check_var('ARCH', 'aarch64')) {
-            push @image_names,  "registry.suse.de/suse/sle-${lowerversion}/docker/update/cr/totest/images/suse/sles${nodashversion}";
-            push @stable_names, "registry.suse.com/suse/sles${nodashversion}";
+            push @untested_images, "registry.suse.de/suse/sle-${lowerversion}/docker/update/cr/totest/images/suse/sles${nodashversion}";
+            push @official_images, "registry.suse.com/suse/sles${nodashversion}";
         }
     }
     elsif (is_sle(">=15", $version) && is_released) {
         my $lowerversion = lc $version;
         # Location for maintenance builds
-        push @image_names,  "registry.suse.de/suse/sle-${lowerversion}/update/cr/totest/images/suse/sle15:${dotversion}";
-        push @stable_names, "registry.suse.com/suse/sle15:${dotversion}";
-    }
-    elsif (is_sle(">=15-sp3", $version)) {
```

Comment:
Why is this deleted?

Reply:
15-SP3 is already released. We should enable 15-SP3 in the bot as well, so we should pull also those builds form the same place as the others.

Reply:
> Fine by me, I will double check once more later. We for sure need verification runs.
> 
> Just please documment the `CONTAINER_IMAGE_TYPE` variable to `variables.md` as well.

I actually was thinking of using it a bit differently... Something like `CONTAINERS_UNTESTED_IMAGES=1` instead of depending on a string... WDYT?

Reply:
Just changed that var. The OpenQA bot is also configured to push `CONTAINERS_UNTESTED_IMAGES=1`

Reply:
Instead of deleting, just bump to SP4?